local_planner/src/nodes/planner_functions.cpp

```diff
@@ -308,13 +308,56 @@ void getCostMatrix(const Histogram& histogram, const Eigen::Vector3f& goal,
     }
   }
 
-  float smoothing_margin_degrees = 45;
+  float smoothing_margin_degrees = 30;
```

Comment:
either make it a parameter or crystal clear that it is a constant.
`const float SMOOTHING_MARGIN_DEG = 30.0f`

Reply:
made it a parameter